tests/constraints/tests.py

```diff
@@ -29,7 +29,7 @@ def test_deconstruction(self):
         self.assertEqual(kwargs, {'check': check, 'name': name})
 
     @skipUnlessDBFeature('supports_table_check_constraints')
-    def test_model_constraint(self):
+    def test_database_constraint(self):
```

Comment:
I think the idea behind the original name is that the constraint is declared on the model (meta). The new name doesn't seem markedly better to me.